### PR TITLE
[CAL-1410] Tokens update - Lightmode issues

### LINKED ISSUE #8089: [CAL-1410] Tokens update - Lightmode issues
* [ ] Shadow & border on public booking page timezone dropdown is gone. Need to check others

* [ ] Toggle "Off" has wrong styling. [Figma](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=309%3A44841&t=30onBbWkRyrGOgzn-1)

- [ ] black "x" in select
CleanShot 2023-04-04 at 22 25 22@2x

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1410](https://linear.app/calcom/issue/CAL-1410/tokens-update-lightmode-issues)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
Fixes #8089 

Loom Video: https://www.loom.com/share/1d13b4bcba0a4c9eac00d22f5a407dae

Could not reproduce black `x` issue

Example:
image

### CODE REVIEW

File: packages/ui/components/form/switch/Switch.tsx [hunk 1]
```diff
@@ -40,17 +40,17 @@ const Switch = (
...
+            "focus:ring-brand-default h-5 w-[34px] rounded-full shadow-none focus:border-neutral-300 focus:outline-none focus:ring-2 focus:ring-neutral-800 focus:ring-offset-1",
             props.className
           )}
           {...primitiveProps}>
           <PrimitiveSwitch.Thumb
             id={id}
             className={cx(
               "block h-[14px] w-[14px] rounded-full transition will-change-transform ltr:translate-x-[4px] rtl:-translate-x-[4px] ltr:[&[data-state='checked']]:translate-x-[17px] rtl:[&[data-state='checked']]:-translate-x-[17px]",
-              isChecked ? "bg-brand-accent shadow-inner" : "bg-default",
+              isChecked ? "bg-default shadow-inner dark:bg-gray-100" : "dark:bg-darkgray-100 bg-white",
```
Comment (PeerRich) [inline packages/ui/components/form/switch/Switch.tsx:53]: we wanna keep brand here

**Comment by gitstart-calcom** [comment]
Hi @PeerRich! PR has been updated, please take a look
https://www.loom.com/share/1d13b4bcba0a4c9eac00d22f5a407dae

File: packages/ui/components/form/timezone-select/TimezoneSelect.tsx [hunk 2]
```diff
@@ -52,18 +52,22 @@ export function TimezoneSelect({
...
             state.isFocused && "bg-subtle",
-            state.isSelected && "bg-emphasis text-default",
+            state.isSelected && "bg-emphasis",
             timezoneClassNames?.option && timezoneClassNames.option(state)
           ),
         placeholder: (state) => classNames("text-muted", state.isFocused && "hidden"),
         dropdownIndicator: () => "text-default",
         control: (state) =>
           classNames(
+            "!cursor-pointer",
```
Comment (zomars) [inline packages/ui/components/form/timezone-select/TimezoneSelect.tsx:70]: What does the bang `!` does in this context?

Comment (sean-brydon) [reply packages/ui/components/form/timezone-select/TimezoneSelect.tsx:70]: ! just adds important to cursor-pointer. Changes are react-select doesnt remove this from their core styles so we need to force it

**Review by PeerRich — APPROVED** [review]
looks good unless @sean-brydon has objections?

**Review by zomars — APPROVED** [review]
Ship it

**Pull request merged**